core/build.gradle.kts

```diff
@@ -25,7 +25,7 @@ kotlin {
             implementation(npm("midi-file", "1.2.4"))
             implementation(npm("js-yaml", "4.1.0"))
             implementation(
-                npm("@sevenc-nanashi/valuetree-ts", "npm:@jsr/sevenc-nanashi__valuetree-ts@0.2.0"),
+                npm("@sevenc-nanashi/valuetree-ts", "npm:@jsr/sevenc-nanashi__valuetree-ts@0.2.1"),
```

Comment:
Is this change causing the build error? maybe the yarn.lock file needs to be updated as well?

Reply:
Oh I haven't pushed my local commit; I'll push later